index.html

```diff
@@ -2491,6 +2677,14 @@ <h3>
             A user agent can present this icon where space constraints and/or
             color requirements differ from those of the application icon.
           </dd>
+          <dt>
+            <dfn data-lt="badge masking">masking</dfn>:
+          </dt>
+          <dd>
+            The image is designed with <a>masking</a> in mind, such that any
```

Comment:
Drop the link (which links to this very sentence) or link up to the masking section above.